kroma-validator/cmd/validator/cmd.go

```diff
@@ -0,0 +1,454 @@
+	if err != nil {
+		return fmt.Errorf("failed to parse ValidatorManager address: %w", err)
+	}
+
+	if err = sendTransaction(ctx, valManagerAddr, txData, "0"); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func Unjail(ctx *cli.Context) error {
+	valManagerAbi, err := bindings.ValidatorManagerMetaData.GetAbi()
+	if err != nil {
+		return fmt.Errorf("failed to get ValidatorManager ABI: %w", err)
+	}
+
+	validatorAddr, err := getValidatorAddress(ctx)
+	if err != nil {
+		return fmt.Errorf("failed to get validator address: %w", err)
+	}
+
+	txData, err := valManagerAbi.Pack("tryUnjail", validatorAddr, false)
+	if err != nil {
+		return fmt.Errorf("failed to create try unjail transaction data: %w", err)
+	}
+
+	valManagerAddr, err := opservice.ParseAddress(ctx.String(flags.ValManagerAddressFlag.Name))
+	if err != nil {
+		return fmt.Errorf("failed to parse ValidatorManager address: %w", err)
+	}
+
+	if err = sendTransaction(ctx, valManagerAddr, txData, "0"); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func ChangeCommissionRate(ctx *cli.Context) error {
+	commissionRate := uint8(ctx.Uint64("commissionRate"))
+
+	valManagerAbi, err := bindings.ValidatorManagerMetaData.GetAbi()
+	if err != nil {
+		return fmt.Errorf("failed to get ValidatorManager ABI: %w", err)
+	}
+
+	txData, err := valManagerAbi.Pack("changeCommissionRate", commissionRate)
+	if err != nil {
+		return fmt.Errorf("failed to create change commission rate transaction data: %w", err)
+	}
+
+	valManagerAddr, err := opservice.ParseAddress(ctx.String(flags.ValManagerAddressFlag.Name))
+	if err != nil {
+		return fmt.Errorf("failed to parse ValidatorManager address: %w", err)
+	}
+
+	if err = sendTransaction(ctx, valManagerAddr, txData, "0"); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func sendTransaction(ctx *cli.Context, txTo common.Address, txData []byte, txValue string) error {
+	txMgrConfig := txmgr.ReadCLIConfig(ctx)
+	txManager, err := txmgr.NewSimpleTxManager("validator-balance", log.New(), &metrics.NoopTxMetrics{}, txMgrConfig)
+	if err != nil {
+		return fmt.Errorf("failed to create tx manager: %w", err)
+	}
+
+	value, success := new(big.Int).SetString(txValue, 10)
+	if !success {
+		return errors.New("failed to parse tx value")
+	}
+
+	txCandidate := txmgr.TxCandidate{
+		TxData:   txData,
+		To:       &txTo,
+		GasLimit: 0,
+		Value:    value,
+	}
+
+	_, err = txManager.Send(context.Background(), txCandidate)
+	if err != nil {
+		return fmt.Errorf("failed to send transaction: %w", err)
+	}
+
+	return nil
+}
+
+func getValidatorAddress(ctx *cli.Context) (common.Address, error) {
+	txMgrConfig := txmgr.ReadCLIConfig(ctx)
+	txManager, err := txmgr.NewSimpleTxManager("validator-balance", log.New(), &metrics.NoopTxMetrics{}, txMgrConfig)
+	if err != nil {
+		return common.Address{}, fmt.Errorf("failed to create tx manager: %w", err)
+	}
+
+	return txManager.Config.From, nil
+}
+
+func getGovTokenAddress(ctx *cli.Context) (common.Address, error) {
```

Comment:
We can simplify using below. Remove this function and just get ASSET_TOKEN in `Approve` function directly.
```go
	assetManagerContract, err := bindings.NewAssetManagerCaller(assetManagerAddr, txManager.Backend.(*ethclient.Client))
```